src/Workspaces/Core/MSBuild/MSBuild/MSBuildProjectLoader.Worker.cs

```diff
@@ -132,72 +132,67 @@ public async Task<ImmutableArray<ProjectInfo>> LoadAsync(CancellationToken cance
                 var results = ImmutableArray.CreateBuilder<ProjectInfo>();
                 var processedPaths = new HashSet<string>(PathUtilities.Comparer);
 
-                _buildManager.StartBatchBuild(_globalProperties);
```

Comment:
why did this go away?

Reply:
Becuase this now happens in the other process.